releasenotes/notes/num-ancilla-qubits-deprecation-d616c3fc24250d22.yaml

```
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    The property ``num_ancilla_qubits`` from the class :class:`~qiskit.circuit.library.arithmetic.PolynomialPauliRotations` 
```

I missed this the first time around, this should be `qiskit.circuit.library.PolynomialPauliRotations` as that's the path in the docs (I often use the hosted docs to check this: https://qiskit.org/documentation/stubs/qiskit.circuit.library.PolynomialPauliRotations.html). Although you can simplify this like you did below with:
```suggestion
    The property ``num_ancilla_qubits`` from the class :class:`.PolynomialPauliRotations` 
```